packages/search-in-workspace/src/browser/search-in-workspace-frontend-module.ts

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright (C) 2017 Erisson and others.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+import { ContainerModule } from "inversify";
+import { SearchInWorkspaceService, SearchInWorkspaceClientImpl } from '../common/search-in-workspace-service';
+import { SearchInWorkspaceServer } from '../common/search-in-workspace-interface';
+import { WebSocketConnectionProvider } from '@theia/core/lib/browser';
+import {
+    QuickSearchInWorkspace,
+    SearchInWorkspaceCommandContribution,
+    SearchInWorkspaceMenuContribution,
+    QuickSearchInWorkspaceKeybindingContribution
+} from './quick-search-in-workspace';
+import {
+    CommandContribution,
+    MenuContribution,
+    KeybindingContribution
+} from "@theia/core";
+
+export default new ContainerModule(bind => {
+    bind(QuickSearchInWorkspace).toSelf().inSingletonScope();
+
+    /* Command contribution */
```

Comment:
Could we have comments only if they really add value?

Reply:
In this case I agree it's not so useful, but in general this is a very subjective topic :)